Formula/global.rb

```diff
@@ -31,6 +32,7 @@ class Global < Formula
   depends_on "python@3.9"
 
   uses_from_macos "ncurses"
+  uses_from_macos "sqlite"
```

Comment:
Actually, now that I think about it, let's make this `depends_on "sqlite"`. We already have a `python@3.9` dependency that `depends_on "sqlite"`, so this basically comes for free anyway.

Reply:
It turns out that sqlite is not really a dependency because global includes its own copy of sqlite (global-6.6.7/libdb/sqlite3.c).

Reply:
Can we get this to use Homebrew `sqlite`? We're usually not too keen on [vendored dependencies](https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae).

Reply:
The included sqlite is built differently (`-DSQLITE_THREADSAFE=0 -DSQLITE_OMIT_LOAD_EXTENSION` [line 22](https://cvs.savannah.gnu.org/viewvc/global/global/libglibc/Makefile.am?revision=1.7&view=markup)), likely for performance reasons.

Reply:
Yes, but I don't think that's a good enough reason to overlook the policy.

Reply:
Just took a look. Manual replacement of the vendored SQLite sources (3.8.7.1, yikes!) with current (3.36.0) seems to build OK, though I don't know if it works. (At the very least, it speaks volumes about Dr. Richard Hipp's commitment to backwards compatibility.)

As for linking with Homebrew `sqlite3`, I'm currently working on a patch that adds the ability to link with external SQLite libraries, without taking away the vendored option. Hopefully, this will make the patch more acceptable to upstream. Stay tuned...